File: server/app/schemas/match.py
from pydantic import BaseModel
from datetime import datetime


class MatchResultResponse(BaseModel):
    id: int
    official_id: int
    jurisdiction_id: int
    agenda_item_id: int | None
    matched_interest: str
    confidence: float
    flagged: bool
    reason: str | None
    pdf_url: str | None
    attachment_name: str | None
    event_date: str | None
    year: int
    created_at: datetime

    class Config:
        from_attributes = True

File: website/src/types/match.ts
export interface Match {
  id: number
  official_id: number
  jurisdiction_id: number
  agenda_item_id: number
  matched_interest: string
  confidence: number
  flagged: boolean
  reason: string | null
  pdf_url: string | null
  attachment_name: string | null
  event_date: string | null
  year: number
  created_at: string
}
